### ISSUE TITLE
Add a staging web server

### ISSUE THREAD

**Issue by gdams** [AdoptOpenJDK/openjdk-website#44]
I'll have a look at whether we can make a jenkins job that can pull changes from a PR and run them on a staging server. If that works we can perhaps get @tellison to point staging.adoptopenjdk.com to that server for easy previewing of changes

**Comment by gdams** [comment]
http://ci.adoptopenjdk.net/job/adoptopenjdk-website-staging/

**Comment by gdams** [comment]
job is running on my aws instance, just waiting on @karianna to provision a permanent machine

**Comment by tellison** [comment]
FYI http://staging.adoptopenjdk.net is set-up to current machine.  Will change it when we get a different home.

**Comment by gdams** [comment]
Thanks

**Comment by gdams** [comment]
It would be nice to add a banner to the staging server website to make it clear that it is for development only, Not sure how it would be best to do this, perhaps inject the html in using the jenkins job?
<img width="1440" alt="screen shot 2017-03-24 at 23 52 50" src="https://cloud.githubusercontent.com/assets/20224954/24317233/2b790696-10ed-11e7-8ef2-28857b997f57.png">

**Comment by karianna** [comment]
Closing this and opening up another issue for the banner
